Mathlib/Analysis/SpecialFunctions/Gamma/BohrMollerup.lean

```diff
@@ -352,6 +356,23 @@ theorem Gamma_strictMonoOn_Ici : StrictMonoOn Gamma (Ici 2) := by
   rw [inter_eq_right]
   exact fun x hx => two_pos.trans_le <| mem_Ici.mp hx
 
+theorem exists_isMinOn_Gamma_Ioi : ∃ x ∈ Ioo 1 2, IsMinOn Gamma (Ioi 0) x := by
```

Comment:
I think I want an even stronger statement. I want uniqueness too. Can you manage that?

Reply:
I think uniqueness requires upgrading the current `ConvexOn` to `StrictConvexOn` and then using `StrictConvexOn.eq_of_isMinOn`. As far as Mathlib knows the Gamma function might be constant on some subset of `[1,2]`, and so the minimum is not unique.

This sounds useful but way beyond the scope of this small PR, I'm not even sure that the same proof technique that works for convexity will work for strong convexity, since it uses Hölder's inequality which isn't strict.

Do you have an idea of how to get strict convexness, or uniqueness of the minimum, or another way you want this statement to be stronger?

Reply:
I'm confused; did you add `awaiting-author` because you expect me to prove that? Prove the strict Hölder iff, and then prove that the conditions for it being strict are met in the case of the gamma function? I think that is unreasonable, unless I'm missing something and that is an easy 5 lines argument, or you're willing to provide substantially more guidance